text/metrics/0238-pipeline-monitoring.md

```diff
@@ -0,0 +1,426 @@
+|----------------|---------------------------------------------|----------------------------|------------------------------------------------------------|
+| `otel.signal`  | Name of the telemetry signal                | Basic (Opt-out)            | `traces`, `logs`, `metrics`                                |
+| `otel.name`    | Type, name, or "type/name" of the component | Normal (Opt-out)           | `probabilitysampler`, `batch`, `otlp/grpc`                |
+| `otel.success` | Boolean: item considered success?           | Normal (Opt-out)           | `true`, `false`                                            |
+| `otel.reason`  | Explaination of success/failures.           | Detailed (Opt-in)          | `ok`, `timeout`, `permission_denied`, `resource_exhausted` |
+| `otel.scope`   | Name of instrumentation.                    | Detailed (Opt-in)          | `opentelemetry.io/library`                                 |
+
+For example, when a sampler processor drops an item it may report
+`success=true`, but when a queue processor drops an item it may report
+`success=false`.
+
+#### SDK default configuration
+
+| Metric name         | Enablement level |
+|---------------------|------------------|
+| `otel.sdk.received` | Basic            |
+| `otel.sdk.dropped`  | Normal           |
+| `otel.sdk.exported` | Detailed         |
+
+#### Collector default configuration
+
+| Metric name               | Enablement level |
+|---------------------------|------------------|
+| `otel.{station}.received` | Basic            |
+| `otel.{station}.dropped`  | Basic            |
+| `otel.{station}.exported` | Basic            |
+
+## Pipeline monitoring as a service
+
+With this specification, users operating OpenTelemetry SDKs and
+Collectors sending to a third-party observability system may expect to
+be provided with information about telemetry losses.
+
+### Configuration of pipeline monitoring metrics
+
+OpenTelemetry SDKs will be configured by default to send pipeline
+monitoring metrics using Meter instances obtained from the global
+meter provider.
+
+SDKs authors SHOULD provide users with an option to configure an
+alternate destination for pipeline monitoring metrics, so that
+pipeline metrics can be monitored independently of ordinary telemetry
+data.
+
+### Inference rules for service providers
+
+By design, losses and failures due to intermediate collectors will be
+observable to the service provider, as long as all original producers
+report pipeline-monitoring metrics.  For all telemetry producers with
+basic-level pipeline monitoring enabled, the telemetry system will be
+able to compare the actual number of OpenTelemetry spans, metric data
+points, and log records received against the numbers that entered the
+system's own pipelines.  In aggregate, the number of items entering
+the pipeline should match the number of items successfully received,
+otherwise the system is capable of reporting the combined losses to
+the user.
+
```

Comment:
This is a vital paragraph to understand why the default, different levels of detail for each station. Putting a end-to-end example, as you did with https://github.com/jmacd/oteps/blob/jmacd/drops/text/metrics/0238-pipeline-monitoring.md#pipeline-monitoring-diagram would work great here (albeit with numbers/metrics instead of charts)